### PR TITLE
Add rhel-8 translation support

### PR DESCRIPTION
**Pull request by jkonecny12**

### CODE REVIEW

**Review by VladimirSlavik — APPROVED** [review]
Looks good to me. Thank you!

File: rhel-8/as.po [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+# Translators:
+# Nilamdyuti Goswami <ngoswami@redhat.com>, 2013-2014
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-05-26 17:19+0200\n"
+"PO-Revision-Date: 2015-01-14 06:43+0000\n"
+"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
+"Language-Team: Assamese (http://www.transifex.com/projects/p/initial-setup/language/as/)\n"
```
Comment (VladimirSlavik) [inline rhel-8/as.po:14]: Is the URL still valid?

Comment (jkonecny12) [reply rhel-8/as.po:14]: That is not my and it will be probably replaced by Weblate after import.

Comment (M4rtinK) [reply rhel-8/as.po:14]: Wow, it that was still there - but yeah, that should get clobbered by Weblate once it starts interacting with the git repo.

**Review by M4rtinK — APPROVED** [review]
Looks good to me, thanks! :)

**Pull request merged**